### PR TITLE
Moises Dobarganes | Portfolio Complete

### PR DESCRIPTION
**Pull request by designerexpert**
Did not use any Frameworks.
Mobile friendly too.

### CODE REVIEW

**Comment by designerexpert** [comment]
Latest Additions:
(1) Projects Section: now has a carousel, that scales with the browser size, and specially in mobile phones. Once it goes down to the size where only 1 item is shown, the Carousel automatically displays the information about that item, otherwise it expects the user to click them for more information.

File: component/nav/nav.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.nav {
+    /* width: 700px; */
```
Comment (ivanmora) [inline component/nav/nav.css:2]: It's best to get rid of commented out code like this. There's always version control and git history if you want to bring something back :)

File: component/section/section.css [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    display: flex;
+    align-items: center;
+    background-color: #ffffff;
+    color: #232323;
+    text-align: justify;
+    justify-content: center;
+    margin-bottom: 20px;
+}
+
+.section__welcome p {
```
Comment (ivanmora) [inline component/section/section.css:55]: With these "section__welcome", "section__about", and such, you're being as specific as an id without using an id. This should really be a modifier on "section". Try to think of the "style" difference when naming class modifiers too. If your styling is really that different between sections (as far as things like font size) then you probably also should be standardizing those too anyways.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    <div class="section section__welcome noselect">
+        <div class="section__header">Welcome <hr/></div>
+        <p>
+            Welcome to my portfolio. Scroll through this page to find a small autobiography. Halfway through there will be some images,
+            descriptions, and links to some of my projects and repositories, and towards the bottom you can find my contact
+            information. Thank you for stopping by. Feel free to browse around and be sure to let me know if there is something
+            you would like me to add.
+        </p>
+    </div>
+    <div class="section__divider" id="about"></div>
```
Comment (ivanmora) [inline index.html:45]: "section__divider" would indicate that this element is a child of a "section" element, but it's not here nested under one.

File: component/sideScroll/sideScroll.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
+window.onload = () => doResize();
+window.onresize = () => doResize();
+
+const doResize =() => {
```
Comment (ivanmora) [inline component/sideScroll/sideScroll.js:4]: Nice

**Review by ivanmora — COMMENTED** [review]
You are doing really great! I think my only constructive feedback right now is that your usage of the BEM convention is not quite right. Now, knowing how to use BEM isn't the most important thing, but I would just say to go back through the rules again, and try to re-understand them. Other than that, things look really great, and I very much appreciate your willingness to help your peers. I see a leader in you, and I want to encourage you to keep up your good spirits, and keep rocking it with the code!

File: component/section/section.css (hunk 2 above)
Comment (designerexpert) [reply component/section/section.css:55]: Oh, i see. I can combine all the similarities on "section", and then only add the individual differences on each separate one like "section_welcome" and so on. Will have some refactoring to do :) . But this just clicked for me. Thanks for the comment!

File: index.html (hunk 3 above)
Comment (designerexpert) [reply index.html:45]: I think for BEM's sake, i will change "section__divider" to just "divider". Thanks again for going through my code.

**Pull request closed without merging**